oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/AbstractDocumentNodeState.java

```diff
@@ -162,4 +162,8 @@ private boolean revisionEquals(AbstractDocumentNodeState other) {
         return this.getLastRevision() != null
                 && this.getLastRevision().equals(other.getLastRevision());
     }
+
+    static void configurePerfLogger(long infoLogMillis) {
```

Comment:
I would prefer a short javadoc to explain what this does

Reply:
Done